packages/strapi-plugin-content-manager/admin/src/containers/App/helpers.js

```diff
@@ -0,0 +1,94 @@
+import { List } from 'immutable';
```

Comment:
Please add comments in the entire file to make it more readable for everyone

Reply:
Done